project/Build.scala

```diff
@@ -319,15 +324,16 @@ object ScaldingBuild extends Build {
 
   lazy val scaldingCommons = module("commons").settings(
     libraryDependencies ++= Seq(
-      "com.backtype" % "dfs-datastores-cascading" % dfsDatastoresVersion,
-      "com.backtype" % "dfs-datastores" % dfsDatastoresVersion,
+      "org.apache.hadoop" % "hadoop-client" % hadoopVersion % "provided",
+      "temp.cchepelov.com.backtype" % "dfs-datastores-cascading3" % dfsDatastoresVersion, // FIXME: https://github.com/nathanmarz/dfs-datastores/pull/53
```

Comment:
@cchepelov  I'm running some tests against that branch, so it should get merged to develop soonish. Go ahead and merge that branch if that keeps this PR moving though. Thanks.